packages/sdk-client/src/domain/domain-helper.ts

```diff
@@ -6,6 +6,11 @@ export const CONVERSATION_TEMPLATES_HOSTNAME = `https://${REGION_PATTERN}templat
 export const ELASTIC_SIP_TRUNKING_HOSTNAME = 'https://elastic-trunking.api.sinch.com';
 export const FAX_HOSTNAME = `https://${REGION_PATTERN}fax.api.sinch.com`;
 export const MAILGUN_HOSTNAME = `https://api.${REGION_PATTERN}mailgun.net`;
+export type MailgunStorageHostname =
```

Comment:
Onto next iteration, an added value to have these enum values coming from generator: they could be shared across SDKs